python/mxnet/contrib/onnx/onnx2mx/_import_helper.py

```diff
@@ -75,6 +75,7 @@
     'LeakyRelu'         : leaky_relu,
     'Elu'               : _elu,
     'PRelu'             : _prelu,
+    'Selu'              : _selu,
```

Comment:
will there be a namespace collision with selu? why _selu?

Reply:
followed the same naming convention as _elu, _prelu.